manifests/s/Samsung/DeX/2.4.1.11/Samsung.DeX.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS0.CRLF.5-1-22621-436.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Samsung.DeX
+PackageVersion: 2.4.1.11
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+Installers:
+- Architecture: x86
+  InstallerUrl: https://downloadcenter.samsung.com/content/SW/202210/20221019170343938/SamsungDeXSetupWin.exe
+  InstallerSha256: 11A2B5ED0E9AA20BFE175A8015BC78FA6EC9D1B2DC7F54EF5334815E7D0A44A6
+  ProductCode: '{A45AB475-00B9-4CCF-A277-6513F2C85C92}'
```

```suggestion
  ProductCode: '{7956e352-c776-4193-bc90-4cc41187c2cd}'
```
WinGet only focuses on ProductCode that doesn't have `SystemComponent = 1`.

This was fetched from Windows Registry: `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{7956e352-c776-4193-bc90-4cc41187c2cd}`

```
Name          : Samsung DeX
Publisher     : Samsung Electronics Co., Ltd.
Version       : 2.4.1.11
ProductCode   : {7956e352-c776-4193-bc90-4cc41187c2cd}
Registry Path : Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\{7956e352-c776-4193-bc90-4cc41187c2cd}
Uninstall     : winget uninstall --id "{7956e352-c776-4193-bc90-4cc41187c2cd}"
```

---

@ItzLevvie thanks, I followed YamlCreate.ps1 suggestion to get the code and it gave me that one.